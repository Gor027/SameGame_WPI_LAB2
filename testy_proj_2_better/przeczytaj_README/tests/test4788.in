 KKK  K((K(K   ( K( (   (( K(K(K  (  KKK   KKKK
(  ((K(K  ( K((( ( (KKK(KKK ( ((( (KK  (  
   (K   K(KKK ((  (K((  K   (KK((K( ((
 ( K(KKK ( KKK( (K
K 
  (((KKK K(KKK
