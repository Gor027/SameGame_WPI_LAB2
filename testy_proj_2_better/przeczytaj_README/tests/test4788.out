   K
   (( K( K(KK
   (KK(K ((KK((( K  (   (   ( (K  (
 KKK(KKK((KKKK(( (((KK  K(  (K((( ( KK K
(((((KKKKK(KKK(((K(K((K(KKKK(KK((K(K((K(KKKK
